build.sh

```diff
@@ -222,7 +222,7 @@ case $__HostOS in
         __monoroot=/usr/local
         ;;
     OSX)
-        __monoroot=/Library/Frameworks/Mono.framework/Versions/Current
+        __monoroot=/usr/local
```

Comment:
I must not be using the default install location, then. I did build mono from source, so that may have something to do with it.